### ISSUE TITLE
If navigation menu is graphical, “alt” text should clearly provide alternative information to meet ADA guidelines [2]

### ISSUE THREAD

**Issue by Justinzenki** [kaygee92/qawebsite#7]
Summary: Navigation menu is not graphical and alt text is not provided

Severity/priority: 3/3

Version: 1.0

Environment: HTML

Expected: Navigation menu to have alt text provided

Actual: Navigation menu does not have alt text provided

Steps to reproduce: View code in sublime text and view all pages including the navigation menu to view

**Comment by kaygee92** [comment]
Fixed the alt in all nav bars
